test/multilocale/diten/oneOrMoreLocales/testHiddenLauncher.prediff

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Test that launcher can find its _real binary when
+# both are buried somewhere in $PATH.
+
+mkdir hidefiles 
+export PATH="`pwd`/hidefiles:$PATH"
+
+basefile=testHiddenLauncher
+if [[ -e $basefile ]] ; then
+  mv $basefile hidefiles
+fi
+
+if [[ -e ${basefile}_real ]] ; then
+  mv ${basefile}_real hidefiles
+fi
+
+$basefile -nl1 > ${basefile}.comp.out.tmp 2>&1
```

Comment:
I think we want an `exec.out.tmp` suffix here instead of `comp.out.tmp`.  The current code will overwrite any output we got during compilation.  While it's admittedly _very_ unlikely we would have any, and it wouldn't be the test's fault if we did, we still don't want to overwrite it.

Reply:
The test is stopped before the execution step, so it expects there to only be a `.comp.out.tmp`. What if I append to the file instead? `$basefile -nl1 >> ${basefile}.comp.out.tmp 2>&1`. That way we don't lose any output if there is any.

Reply:
Oh, right, we've got a `.noexec`.  Yeah, append should work fine.